fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/Constants.java

```diff
@@ -56,4 +56,7 @@ public class Constants {
 
   /** Number of milliseconds in a day */
   public static final long MS_IN_A_DAY = 1000 * 60 * 60 * 24L;
+
+  /** Constants to rotate up or down along the x axis */
+  public static final float ORIENTATION_UP = 0f, ORIENTATION_DOWN = 180f;
```

Comment:
Bit of nitpicking but I am not a fan of multiple variables declaration in one line. It's not great for readability IMO and is not something you'll see a lot in this codebase.

Reply:
Sure I'll split them!